great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -302,9 +302,8 @@ def _build_numeric_columns_rule() -> Rule:
 
         # Step-2: Declare "ParameterBuilder" for every metric of interest.
 
-        column_partition_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.build_partition_parameter_builder(
+        column_histogram_single_batch_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.build_histogram_single_batch_parameter_builder(
             name="column_values.partition",
-            bucketize_data=True,
```

Comment:
My understanding is that previously `bucketize_data=False` would return `column.value_counts`.